src/Core/Checkout/Cart/Order/OrderConversionContext.php

```diff
@@ -16,8 +17,13 @@ class OrderConversionContext extends Struct
 
     protected bool $includeTransactions = true;
 
+    /**
+     * @deprecated tag:v6.8.0 - will be removed, use `includePersistentData` instead
+     */
     protected bool $includeOrderDate = true;
 
+    protected bool $includePersistentData = true;
```

Comment:
Not sure if that name is a good description. If we cannot find a better name, I propose adding a commet

Reply:
any opinions, @Bird87ZA @lernhart ?

Reply:
How about `ImmutableData` ?

I'm taking the credit for that, Max

Reply:
I hate the `shouldInclude` naming scheme

Maybe sth. like `preserveMetadata`?

Reply:
Metadata is also a good option, yes. Is your issue with the `should` or the `include`? Because I think, include is more fitting than preserve. Preserve would imply for me that stuff is being protected, but it's simply not included in the array in the `CartTransformer`.

Reply:
internal poll in our team channel says `includePersistentData`. So no change.